src/core/generators/react-query.ts

```diff
@@ -69,7 +67,9 @@ const generateAxiosFunction = (
           )?.slice(1, -1)} ...options}`
         : '// eslint-disable-next-line\n// @ts-ignore\n options'
       : '';
-
+   
+    const mutatorTyping = `<Data extends unknown ? ${ response.definition || 'Data' } : Data>`;
```

Comment:
In that case, I prefer the following solution which remove the conditional type and replace it only with the generic

```suggestion
    const mutatorTyping = response.definition ? `<Data extends unknown ? ${ response.definition} : Data>` : 'Data';
```